Mathlib/Data/ZMod/Basic.lean

```diff
@@ -981,6 +987,14 @@ theorem val_eq_zero : ∀ {n : ℕ} (a : ZMod n), a.val = 0 ↔ a = 0
 theorem val_ne_zero {n : ℕ} (a : ZMod n) : a.val ≠ 0 ↔ a ≠ 0 :=
   (val_eq_zero a).not
 
+theorem val_pos_of_ne_zero {n : ℕ} {a : ZMod n} (h : a ≠ 0) : 0 < a.val :=
+  Nat.pos_of_ne_zero <| (val_ne_zero a).mpr h
```

Comment:
Surely
```suggestion
theorem val_pos {n : ℕ} {a : ZMod n} : 0 < a.val ↔ 0 < a:= by simpa [pos_iff_ne_zero] using val_ne_zero _
```

Reply:
To my knowledge, there's no `LT` instance on `ZMod n` and I think that's a conscious choice

Reply:
Then
```suggestion
theorem val_pos {n : ℕ} {a : ZMod n} : 0 < a.val ↔ a ≠ 0 := by simpa [pos_iff_ne_zero] using val_ne_zero _
```